chapters/api/content/api.md

```diff
@@ -0,0 +1,92 @@
+# Exposing and Querying Smart Contract Data on MultiversX
+
+This lab outlines the architecture and methods for fetching real-time and historical data related to the MultiversX network.
+
+## 🎯 Objectives
+
+By the end of this session, you will be able to:
+
+* Understand how **Observers** facilitate **VM Queries** without participating in consensus.
+* Navigate the **ElasticSearch** indices to find specific data fields.
+* Differentiate between the **Gateway (Proxy)** and the **API** layer.
+* Use the API to perform advanced filtering on transactions and account data.
+
+---
+
+## 1. Real-Time State (VM Queries) & Observers
+
+As experimented with in the previous **dApps** labs, interacting with a Smart Contract doesn't always require a gas-consuming transaction.
+
+To read the **current state** of a contract (e.g., a counter value, a storage variable), we use **VM Queries**.
+
+### How it works: The Role of Observers
+
+The MultiversX network consists of **Validators** (who participate in consensus and produce blocks) and **Observers**.
+
+* **Observers** are nodes that **do not participate in consensus** (they don't produce blocks).
+* However, they execute every transaction to stay **in sync** with the network state.
+* Because they hold the full state of the blockchain locally, they offer an endpoint to execute **Read-Only** transactions.
+
+When you perform a VM Query (via SDK or dApp), you are asking an Observer to simulate the execution of a function locally. The Observer runs the code, retrieves the data from its local storage, and returns the result immediately.
+
+> **Note:** This action allows you to see the "live" state of the contract without broadcasting a transaction to the whole network.
+
+---
+
+## 2. Data Indexing with ElasticSearch
+
+While VM Queries are perfect for *current* data, they cannot easily search through history. For this, MultiversX uses an off-chain indexing solution based on **ElasticSearch**.
```

Comment:
Provide an example here because "search through history" is not intuitive. Maybe someone wants to retrieve his balance on 5th July, 2022.